functions/transformations/percentile_test.go

```diff
@@ -374,6 +374,33 @@ func TestPercentileSelector_Process(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name:     "empty",
+			quantile: 0.5,
+			data: []flux.Table{&executetest.Table{
+				KeyCols:   []string{"t1"},
+				KeyValues: []interface{}{"a"},
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+					{Label: "t1", Type: flux.TString},
+					{Label: "t2", Type: flux.TString},
+				},
+				Data: [][]interface{}{},
+			}},
+			want: []*executetest.Table{{
+				KeyCols: []string{"t1"},
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+					{Label: "t1", Type: flux.TString},
+					{Label: "t2", Type: flux.TString},
+				},
+				Data: [][]interface{}{
+					{nil, nil, "a", nil},
+				},
+			}},
+		},
```

Comment:
Would be good to have a test with null values in the `_value` column in a non-empty input table.

Reply:
Done.